### DISCUSSION TITLE
How to choose 32F105

### DISCUSSION THREAD

**Discussion by vitecd** [stm32duino/Arduino_Core_STM32#2492] [Variants] [answered]
I can see 32F105 variants in the source code but it's not available to select from the Arduini IDE. What I'm doing wrong, please?

**Comment by fpistm** [comment, marked as answer]
Nothing. There is no STM32F105xx declared. If you want to add it feel free to provide a PR following this wiki:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29
